frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx

```diff
@@ -160,11 +155,23 @@ export const OperatorHubList: React.FC<OperatorHubListProps> = ({
             createdAt,
             support,
             capabilities: capabilityLevel,
+            [OperatorHubCSVAnnotationKey.tokenAuthAWS]: tokenAuthAWS,
             [OperatorHubCSVAnnotationKey.actionText]: marketplaceActionText,
             [OperatorHubCSVAnnotationKey.remoteWorkflow]: marketplaceRemoteWorkflow,
             [OperatorHubCSVAnnotationKey.supportWorkflow]: marketplaceSupportWorkflow,
           } = currentCSVAnnotations;
 
+          // TODO remove lodash and implement using Array.prototye.reduce
+          const infraFeatures = _.uniq(
+            _.compact(_.map(parsedInfraFeatures, (key) => InfraFeatures[key])),
+          );
+
+          // FIXME: this is a temporary hack and should be fixed as part of
+          // a refactor to include the new style of infrastructure features
```

Comment:
for reviewers: the new annotation format is tracked in https://issues.redhat.com/browse/PORTENABLE-525

Reply:
Including the link to that Jira issue in the comment would be helpful.